### DISCUSSION TITLE
Use world instead MPI_COMM_WORLD in pair_nnp.cpp

### DISCUSSION THREAD

**Discussion by NewSigma** [deepmodeling/deepmd-kit#367] [General]
I probably found a bug of USER-DEEPMD while using deepmd model. When I start a program A(linked to LAMMPS) using mpi. MPI_COMM_WORLD will broadcast data to all processes of program A rather than the processes of LAMMPS only, which may stuck the program. Simply replace MPI_COMM_WORLD with LAMMPS_NS::Pointers::world should fix it.

If I were wrong, sorry to waste your time.

My environment:
CentOS 6
GCC 7.5.0
lammps-29Oct20
deepmd-kit 1.3.1
tensorflow 1.14.0

**Comment by amcadmus** [comment]
Please refer to #92 and let us know if the reply solves your problem.
